File: 3872-find-most-frequent-vowel-and-consonant/find-most-frequent-vowel-and-consonant.cpp
class Solution {
public:
    int maxFreqSum(string s) {
        vector<char> vowels = {'a', 'e', 'i', 'o', 'u'};
        unordered_map<char, int> freq;
        int n = s.size();
        int vmax = 0, cmax = 0;
        for(auto& i: s) freq[i]++;
        for(auto& i: s){
            if(find(vowels.begin(), vowels.end(), i)!=vowels.end()){
                vmax = max(vmax, freq[i]);
            }
            else cmax = max(cmax, freq[i]);
        }
        return vmax + cmax;
    }
};

File: 3872-find-most-frequent-vowel-and-consonant/find-most-frequent-vowel-and-consonant.py
class Solution:
    def maxFreqSum(self, s: str) -> int:
        vmax, cmax = 0, 0
        freq = {}
        vowels = {'a', 'e', 'i', 'o', 'u'}
        for i in s:
            freq[i] = freq.get(i, 0) + 1
        for i in s:
            if i in vowels:
                vmax = max(vmax, freq[i])
            else : cmax = max(cmax, freq[i])
        return vmax+cmax